cmd/argo/commands/cron/get.go

```diff
@@ -106,10 +106,10 @@ func getCronWorkflowGet(cwf *wfv1.CronWorkflow) string {
 	if len(cwf.Spec.WorkflowSpec.Arguments.Parameters) > 0 {
 		out += fmt.Sprintf(fmtStr, "Workflow Parameters:", "")
 		for _, param := range cwf.Spec.WorkflowSpec.Arguments.Parameters {
-			if param.Value == nil {
+			if !param.HasValue() {
```

Comment:
I don't think this is correct.

HasValue will return true if ValueFrom is set, but GetValue will one return ""